src/tracks/playabletrack/wavetrack/ui/ClipPitchAndSpeedButtonHandle.cpp

```diff
@@ -36,7 +36,7 @@ wxString GetPitchShiftText(int clipCentShift)
       pitchShiftText = wxString::Format("%.2f", std::abs(clipCentShift) / 100.);
       while (pitchShiftText.EndsWith("0"))
          pitchShiftText.RemoveLast();
-      if (pitchShiftText.EndsWith("."))
+      if (pitchShiftText.EndsWith(".") || pitchShiftText.EndsWith(","))
```

Comment:
perhaps a regex non-match for [0-9] would be more complete, but I can't actually find any language using something other than period or comma for a separator, so this probably works too.